### DISCUSSION TITLE
Error when i run testbed.exe

### DISCUSSION THREAD

**Discussion by Slyger** [NVlabs/instant-ngp#352] [Q&A]
error: must specify either a mode or a scene

Probably an easy fix but Idk what I'm doing all that well.

**Comment by Slyger** [comment]
Got it working but when i tried doing testbed.exe -s C:\Users\Alext\Downloads\Teapot\Teapot.obj, i got 

11:46:41 SUCCESS  Loaded mesh "C:\Users\Alext\Downloads\Teapot\Teapot.obj" file with 1 shapes.
11:46:41 SUCCESS  Built TriangleBvh: nodes=245
11:46:41 WARNING  OptiX was not built. Falling back to slower TriangleBVH::ray_intersect.
11:46:41 SUCCESS  Built TriangleOctree: depth=10 nodes=149086 dual_nodes=601331. Populating dual nodes...
11:46:42 SUCCESS  Loaded mesh: triangles=552 AABB=[min=[-45.6151,-0.615103,-30.6151], max=[52.0595,47.8651,30.6151]] after scaling=[min=[0.0062975,0.258125,0.192858], max=[0.993703,0.741875,0.807142]]
11:46:42 ERROR    Network config path configs\sdf\base.json does not exist.

**Comment by leventt** [comment]
you will need to have a folder called configs and a sub directory called sdf/nerf etc with the base.json file which defines parameters for the network/encoding/learning etc.

easiest may be to make sure you run this testbed.exe file under the root directory of the repo (where these configurations exists in the expected folder structure relative to repo's root) where you cloned it, rather than perhaps in the build folder.

**Comment by palastrazas** [comment]
Hi;
When I try to run testbed program I receive similar "ERROR Network config path configs\sdf\base.json does not exist." error message. I was check the  "configs\image\" folder the "base.json" file is exist. How can I fix the this problem. thanks for your help.



![Erorr](https://user-images.githubusercontent.com/29087963/160246598-e82b40c6-8324-4aad-9050-1ed592577be9.jpg)
